aspects/electrical/TripLogic/GunnsTripLogic.cpp

```diff
@@ -76,7 +76,7 @@ bool GunnsTripLogic::checkForTrip(GunnsBasicLink::SolutionResult& result,
     ///   Checks for trips is also skipped if already tripped or the trip limit or converged step
     ///   priority values haven't been initialized.
     if (mMalfInhibitTrip) resetTrip();
-    if (not (mIsTripped or mMalfInhibitTrip or 0.0 == mLimit or 1 > mPriority)) {
+    if (not (mIsTripped or mMalfInhibitTrip or 0.0 == static_cast<double>(mLimit) or 1 > mPriority)) {
```

Comment:
I think it would be simpler to specify 0.0F instead of promoting mLimit here, now that you've shown me about the 0.0F thing :)